docker-compose.yml

```diff
@@ -1,10 +1,8 @@
 services:
   oot:
+    build:
+      dockerfile: Dockerfile
     volumes:
-      - oot-sync:/oot:nocopy
-    image: "oot:latest"
+      - ./:/oot
```

Comment:
From a security standpoint, I would rather not give the container full access to the current directory. My suggestion would be to instead specify a directory and mount only the necessary files.

Reply:
I don't disagree, but if I'm not mistaken, this change is equivalent to the current instructions, just more automated.

Reply:
That's a fair point. I don't personally use the Docker setup, but based on my experience working with container files in other situations, my comment was more of a suggestion to follow best practices than a strict requirement since the change was being introduced. But I can see the downside of imposing a strict directory structure as well.